### PR TITLE
[pt-br] Add docs/reference/glossary/cadvisor.md

### PR DESCRIPTION
**Pull request by MrErlison**
Add
- pt-br/docs/reference/glossary/cadvisor.md

/cc @stormqueen1990 @devlware

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/cadvisor.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: cAdvisor
+id: cadvisor
+date: 2021-12-09
+full_link: https://github.com/google/cadvisor/
+short_description: >
+  Ferramenta que provê o entendimento do uso de recursos e características de desempenho para contêineres
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/cadvisor.md:7]: I like "fornece uma compreensão" better than "provê o entendimento", like you wrote here:
https://github.com/kubernetes/website/blob/86d8e75224ee876c7989fbd5b7375fa5ae6fb157/content/pt-br/docs/reference/glossary/cadvisor.md#L12

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @MrErlison! 👋 

Thank you very much for your contribution! 🎉 

Just left one comment about the construction of the summary sentence. Other than that it LGTM.

**Comment by devlware** [comment]
Thank you @MrErlison.

/approve

**Pull request merged**